pages/common/guetzli.md

```diff
@@ -0,0 +1,15 @@
+# guetzli
+
+> JPEG image compression utility.
+
+- Compress a JPEG image:
+
+`guetzli {{input.jpg}} {{output.jpg}}`
+
+- Create compressed JPEG image from PNG image:
+
+`guetzli {{input.png}} {{output.jpg}}`
+
+- Compress a JPEG image with desired visual quality (84-100):
```

Comment:
It looks like you can't actually use a value below 84 by default.

```
Guetzli should be called with quality >= 84, otherwise the
output will have noticeable artifacts. If you want to
proceed anyway, please edit the source code.
Guetzli processing failed
```

Reply:
Gotcha